presto-main/src/main/java/io/prestosql/operator/scalar/DateTimeFunctions.java

```diff
@@ -392,6 +393,16 @@ public static long dayOfWeekFromDate(@SqlType(StandardTypes.DATE) long date)
         return DAY_OF_WEEK.get(DAYS.toMillis(date));
     }
 
+    @Description("Day of the week of the given date based on start day of the week")
```

Comment:
I am not clear what the second parameters is supposed to do?

Reply:
You can provide start day of the week(1 to 7) and based on this the day of your given date is decided. By default, start day of the week is Monday
Ex : 2020-01-01 is Wednesday assuming your start day is Monday, but if start day is Tuesday then 2020-01-01 will be Thursday.

Reply:
This kind of information needs to be added to the docs. Maybe even with the example.